app/models/katello/repository.rb

```diff
@@ -1048,6 +1054,14 @@ def remove_docker_content(manifests)
       DockerMetaTag.cleanup_tags
     end
 
+    def allow_updates?
+      root.allow_updates
+    end
+
+    def prevent_updates
+      root.prevent_updates
+    end
```

Comment:
Cool Rails trick: If you add a line to the top of this like

```rb
delegate :allow_updates?, :prevent_updates, to: :root 
```

it's equivalent to this code you have here. There's tons of examples in Katello so I would lean towards using it.

Reply:
Appreciate this trick. I implemented it for `prevent_updates` and it works great :)